src/pages/Game.tsx

```
@@ -0,0 +1,530 @@
+        
+        let cellClass = 'cell'
+        if (isTemple) cellClass += ' temple'
+        if (isValidMove) cellClass += ' valid-move'
+        if (isSelectedPiece) cellClass += ' selected-piece'
+
+        cells.push(
+          <div 
+            key={`${row}-${col}`} 
+            className={cellClass}
+            role="button"
+            tabIndex={0}
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)